2_adventures/001/kotp/04_automated_testing_example/lib/fizz_buzz.rb

```diff
@@ -0,0 +1,19 @@
+module FizzBuzz
+  def self.play(number)
+    result = ''
+    result << 'Fizz' if number.divisible_by?(3)
+    result << 'Buzz' if number.divisible_by?(5)
+    result.empty? ? number : result
+  end
+end
+
+module FB_Tools
```

Comment:
In this context someone would understand the name for the module but I think it's a bit generic and adds complexity and also clutter. The `Fixnum` class below is the only place where this module is mixed-in so I think this abstraction here isn't necessary.